package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "react-2020",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack-dev-server --open",
+    "build": "webpack",
+    "build-prod": "set NODE_ENV=production && webpack",
```

Comment:
1. we usually don't need build for dev, so it's fine to leave only one command and run it in production mode
2. this option don't set NODE_ENV var to production (at least on mac and linux)

Reply:
1. don't agree, usually use build in development mode for dev & qa environment 
2. change to using cross-env

Reply:
You will see) But anyway it's not related to the task requirements, so lets merge this pr.